src/org/exist/util/serializer/AbstractSerializer.java

```diff
@@ -30,7 +30,7 @@
         new HTML5Writer()
     };
 
-    private final static Properties defaultProperties = new Properties();
+    protected final static Properties defaultProperties = new Properties();
```

Comment:
The defaultProperties are only used if no other set of properties has been specified and they are not modified. If you call setOutput, you will pass in your own properties.